git_machete/docs.py

```diff
@@ -329,6 +329,12 @@
           Any existing annotations are overwritten for the branches that have an opened PR; annotations for the other branches remain untouched.
           Equivalent to `git machete anno --sync-github-prs`.
 
+        <b>`checkout-pr` [PR_number]`:</b>
+
+          Checks out given pull request (by number) locally under <PR_number> branch. Note this branch is read only, so there is no possibility to push changes there.
```

Comment:
Yeah, as commented on PR, that's the problem with `refs/pull/<PR>/head` branches... TBH they're of very limited usefulness :/ I'd rather check them out as regular local branches instead